org.jacoco.build/pom.xml

```diff
@@ -394,6 +405,11 @@
           <artifactId>xml-maven-plugin</artifactId>
           <version>1.0</version>
         </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>exec-maven-plugin</artifactId>
+          <version>1.6.0</version>
```

Comment:
@marchof so far our lower bound of Maven version for build of project - is 3.0, which in his turn has Java 5 as lower bound, while exec-maven-plugin version 1.6.0 requires Java 7. This explains failure in Travis where we use JDK 6 to run Maven.